apps/outlook-rooms-addin/src/tests/room-booking.component.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { RoomBookingComponent } from '../app/rooms/room-booking.component';
+import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
+import {
+    FormsModule,
+    ReactiveFormsModule,
+    FormBuilder,
+    Validators,
+} from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { BookModule } from '../app/rooms/book.module';
+import { Router } from '@angular/router';
+import { EventFormService } from '@placeos/events';
+import { mockForm, mockRouterStub } from './test-mocks';
+
+describe('RoomBookingComponent', () => {
+    const formModel = mockForm;
+    const fb = new FormBuilder();
+    const form = fb.group(formModel);
+
+    const RouterStub = mockRouterStub;
+    let spectator: Spectator<RoomBookingComponent>;
+
+    const createComponent = createComponentFactory({
+        component: RoomBookingComponent,
+        imports: [
+            ReactiveFormsModule,
+            FormsModule,
+            MatFormFieldModule,
+            BookModule,
+        ],
+        providers: [
+            {
+                provide: Router,
```

Comment:
You shouldn't have to mock routing as `@ngneat/spectator` provides `createRoutingFactory` and `SpectatorRouting` types for this.